adam-core/src/main/scala/org/bdgenomics/adam/rdd/contig/NucleotideContigFragmentRDD.scala

```diff
@@ -490,4 +490,7 @@ sealed abstract class NucleotideContigFragmentRDD extends AvroGenomicRDD[Nucleot
       (p._1, p._2: java.lang.Long)
     }).toJavaRDD()
   }
+
+  override def toString = "%s with %d reference sequences"
```

Comment:
You should be able to factor this into the `GenomicRDD` trait, no?

Reply:
I think it is possible ... but I read some odd things about toString on traits and figured the sealed abstract classes would be the best place.  Then the messages are slightly different so it isn't exactly breaking DRY.

Reply:
What odd things did you see?